### ISSUE TITLE
Study Feedback: Admin-Facing Aspects

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#964]
# TL;DR
If we implement a system for families to give feedback on studies (see issue https://github.com/lookit/lookit-api/issues/963), we need a way for Lookit staff to see and interact with the data. 

# Narrative
As a Lookit admin, I want to be able to easily see information about the feedback we receive from parents (once something like issue 963 is implemented). There are THREE aspects of this: 

1. I can go to any study and see its metrics and all written feedback. Metrics might include the number of 1-5 ratings it has received (e.g., 14 ratings), a histography (or list) of the number of ratings it has received (e.g., 4 ratings of 3, 10 ratings of 2), and the mean rating (e.g., 2.29). 
2. I can get summary statistics for ALL studies that have received at least one rating (e.g., a page where I see, or a way to download a .csv that contains, a row for each study, and columns for name, link to study, PI, date of first review, date of last review, total number of reviews, each of the metrics listed in the first point above, and the number of text comments; PERHAPS a giant string of all the comments as well so that all information is available in one place)
3. I get a notice about studies that might require our attention. We might calibrate this over time, but one way to set it up to begin with is that each week I get an email with a list of all studies with ratings, their mean ratings, and text comments for the worst two studies.

# Implementation Notes
There are lots of potential details to work out here, but I'll expand on the third point above as the one that might be least clear. I am imagining the following email, and notice that the studies are ordered by rating AND comments are included for the two lowest-rated ones. Names are fake.

This is your weekly email about all the studies with ratings, and the comments from the two lowest-rated ones!

4.10 (7 ratings) Fun Dot Study
3.90 (25 ratings) Help Doggie Listen
3.50 (2 ratings) Learning Japanese
2.60 (10 ratings) Help Me Find It!
2.10 (14 ratings) Do You See It?

Comments for Help Me Find It!
- This was too long
- we didn't know how long this would be when we started
- boring
- she said she didn't want to do any more ever when she finally finished

Comments for Do You See It?
- gross pictures
- he was really scared of the dead dog picture
- definitely should remove from your website
- BAD
- I don't trust your website if you have things like this on it, though I like that I could tell you about it here, but it should not have been on website!
